buildenv/jenkins/common/test

```diff
@@ -96,6 +96,12 @@ def publish() {
             junit allowEmptyResults: true, keepLongStdio: true, testResults: '**/junitreports/**/*.xml'
             step([$class: 'Publisher', reportFilenamePattern: '**/testng-results.xml'])
             step([$class: "TapPublisher", testResults: "**/*.tap"])
+            withEnv(["WORKSPACE=${WORKSPACE}", "currentBuild=${currentBuild}"]) { 
```

Comment:
You shouldn't need this line

Reply:
ok, thanks, and yes, wondered as they are both global...